src/onboarding/onboarding.js

```diff
@@ -255,7 +251,7 @@ function addCallbackParticipantPut({
     headers.append('FSPIOP-Source', fspiopSource);
     headers.append('Authorization', `Bearer ${authToken}`);
 
-    const fullCallbackUrl = `${dfspCallbackUrl}/participants/${dfspPartyIdType}/${dfspPartyId}`;
+    const fullCallbackUrl = `${dfspCallbackUrl}/participants/{{partyIdType}}/{{partyIdentifier}}`;
```

Comment:
you removed the params for `dfspPartyId` and `dfspPartyIdType`, but didn't add `partyIdType` and `partyIdentifier`. I think these will be `undefined`.

Reply:
No @Kalaston, these are interpreted by the server as templates. The server fills them in when it creates the callback URL's. That's what has been explained to me by @shashi165.